### PR TITLE
comment corrections

### PR DESCRIPTION
**Pull request by Anderac**
------------------------------------------------------------------------------------------------------------

**Context: I made corrections of typos in comments in code.**

**Description of the Change: I made spelling and grammar correction in comments. I am unfamiliar with making pull request, and I did not make any changes to the changelog-dev.md file because I am unsure which category to use. Likewise, the requirement to run tests did not seem to apply to comment-only changes.**

**Benefits: removal of typos**

**Possible Drawbacks: None**

**Related GitHub Issues: #821**

### CODE REVIEW

**Comment by AmintorDusko** [comment]
Hey @Anderac, thank you for your nice contribution.
You can add your changelog entry to the Improvements section.

**Review by dwierichs — COMMENTED** [review]
Thanks a lot for your contribution and catching those @Anderac! :tada: 
Just leaving some minor comments.

File: pennylane/_grad.py [hunk 1]
```diff
@@ -288,7 +288,7 @@ def circuit(x, y):
     >>> qml.math.shape(jac)
     (3, 2, 4)
 
-    As expected, the tuple was unpacked and we directly received the Jacobian of the
+    As expected, the tuple was unpacked, and we directly received the Jacobian of the
```
[the remark this answers is not in the document]
Comment (trbromley) [reply pennylane/_grad.py:291]:
```suggestion
    As expected, the tuple was unpacked and we directly received the Jacobian of the
```
Yes I have a mild preference for the previous way.

**Review by trbromley — COMMENTED** [review]
Thanks @Anderac for this PR! Sorry for not getting back to you sooner. These changes look great, please could you add an entry to the changelog as well optionally your name in the list of contributors.

File: doc/releases/changelog-dev.md [hunk 2]
```diff
@@ -91,6 +91,9 @@
 
 <h3>Documentation 📝</h3>
 
+* Corrected some spellings and typos scattered throughout the docstrings.
+  [(#4487)](https://github.com/PennyLaneAI/pennylane/pull/4487)
+
```
Comment (trbromley) [inline doc/releases/changelog-dev.md:96]: @Anderac, let us know if you'd like your name added to the list of contributors. I'm planning to merge this PR next week.

**Review by trbromley — APPROVED** [review]

**Comment by dwierichs** [comment]
Hi @Anderac,
thanks again for your contribution here. We will be closing this PR, but we're still happy to incorporate your changes. So if you are interested, feel free to re-open the PR (or open a new one, if that is easier given the merge conflicts).
Thanks!

**Pull request closed without merging**